<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QScrollBar:vertical {
      border: none;
 	  background-color: rgba(255, 255, 255);
      width: 4px;
      margin: 0px 0 0px 0;
  }
  QScrollBar::handle:vertical {
	 border: none;
     border-radius:2px;
      background: rgba(154, 154, 154, 150);
      min-height: 15px;
  }
QScrollBar::sub-line:vertical {
      border: none;
      height: 0px;
      subcontrol-position: top;
      subcontrol-origin: margin;
  }
QScrollBar::add-line:vertical {
      border: none;
      height: 0px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
  }
  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	border:none;
      width: 0px;
      height: 0px;
  }
  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }

</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>242</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Back">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/ICON/picture/Back.ico);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="FriendAccount">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Shop">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/ICON/picture/home_128px_1149880_easyicon.net.ico);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>205</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="GetUp">
             <property name="styleSheet">
              <string notr="true">border-image: url(:/ICON/sucai/add_128px_1206665_easyicon.net.ico);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="MessagelineEdit">
             <property name="styleSheet">
              <string notr="true">border:none;
border-bottom:1px solid #000;
background-color: #F0F0F0;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SendMessage">
             <property name="styleSheet">
              <string notr="true">border-image: url(:/ICON/sucai/send_email_128px_1190680_easyicon.net.ico);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>59</y>
          <width>201</width>
          <height>255</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton_3">
           <property name="styleSheet">
            <string notr="true">border-style:none;padding:10px;border-radius:5px;color:#1ABC9C;background:#E6F8F5;</string>
           </property>
           <property name="text">
            <string>设置备注和标签</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="Arealabel">
           <property name="text">
            <string>地区      陕西西安</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2" colspan="2">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QLabel" name="Remarklabel">
             <property name="text">
              <string>大傻逼</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="FriendAccountlabel">
             <property name="text">
              <string>账号：250</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="Nicknamelabel">
             <property name="text">
              <string>NickNamelabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="3">
            <widget class="QLabel" name="HeadPicture">
             <property name="styleSheet">
              <string notr="true">border-image: url(:/美工按钮专用图/我.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0" rowspan="2" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="styleSheet">
              <string notr="true">border-style:none;padding:10px;border-radius:5px;color:#1ABC9C;background:#E6F8F5;</string>
             </property>
             <property name="text">
              <string>发布信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SendMessageBt">
             <property name="styleSheet">
              <string notr="true">border-style:none;padding:10px;border-radius:5px;color:#1ABC9C;background:#E6F8F5;</string>
             </property>
             <property name="text">
              <string>发送消息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AddFriendBt">
             <property name="text">
              <string>添加好友</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>171</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="SearchFriendAccount">
           <property name="styleSheet">
            <string notr="true">border:none;
border-bottom:1px solid #000;
background-color: #F0F0F0;</string>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>请输入您要添加的好友</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SearchFriendBt">
           <property name="text">
            <string>搜索:张蕊</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
